modules/ROOT/pages/neo4j-drivers.adoc

```diff
@@ -16,14 +16,16 @@
 
 == Community Drivers
 
-** Ruby: link:https://github.com/neo4jrb[neo4j.rb]
+** C: link:https://github.com/majensen/libneo4j-omni[libneo4j-omni]
+
+** Elixir: link:https://github.com/sagastume/boltx[Boltx]
+
+** Perl: link:https://github.com/majensen/perlbolt[Neo4j::Bolt]
 
 ** PHP: link:https://github.com/neo4j-php/[neo4j-php]
 
+** Ruby: link:https://github.com/neo4jrb[neo4j.rb]
+
 ** Rust: link:https://github.com/neo4j-labs/neo4rs[neo4rs]
 
 ** Perl: link:https://github.com/johannessen/neo4j-driver-perl[Neo4j::Driver]
```

Comment:
If you include the Perl modules here, they should probably be listed next to each other.

(Nice idea to add the community drivers, BTW.)

Reply:
Thanks. I messed that one up :sweat_smile: